mathics/core/symbols.py

```diff
@@ -454,6 +471,18 @@ def get_head_name(self):
     def has_symbol(self, symbol_name) -> bool:
         return self.name == ensure_context(symbol_name)
 
+    @property
+    def is_literal(self) -> bool:
+        """
+        True if the value can't change, i.e. a value is set and it does not
+        depend on definition bindings. That is why, in contrast to
+        `is_uncertain_final_definitions()` we don't need a `definitions`
+        parameter.
+
+        Here, we have to be pessimistic and return True.
```

Comment:
here, we return ``False``

Reply:
Thanks  - the docstring has been corrected.